packages/shared/src/utils/deprecated.ts

```diff
@@ -29,7 +29,7 @@ export const deprecated = (fnName: string, warning: string, key?: string): void
   displayedWarnings.add(messageId);
 
   console.warn(
-    `DEPRECATION WARNING: "${fnName}" is deprecated and will be removed in the next major release.\n${warning}`,
+    `DEPRECATION WARNING: Clerk "${fnName}" is deprecated and will be removed in the next major release.\n${warning}`,
```

Comment:
Maybe something like this or similar?
```suggestion
    `Clerk - DEPRECATION WARNING: "${fnName}" is deprecated and will be removed in the next major release.\n${warning}`,
```

Reply:
If we'd want we could get fancy and use something like https://github.com/GoogleChrome/workbox/blob/v7/packages/workbox-core/src/_private/logger.ts

This relates to https://developer.mozilla.org/en-US/docs/Web/API/console#styling_console_output

So we could do:

```js
console.warn('%cclerk', ...styles, 'MESSAGE')
```

Reply:
I do like the styles

Reply:
Let's open a discussion in Slack about theming our logs in general and implement it as different PRs.

Reply:
@nikosdouvlis i have updated the formatting of the warnings. PTAL